ios/Podfile

```diff
@@ -24,5 +24,6 @@ target 'ItaliaApp' do
   use_flipper!()
   post_install do |installer|
     react_native_post_install(installer)
+    __apply_Xcode_12_5_M1_post_install_workaround(installer)
```

Comment:
Is this really needed?

Reply:
This is required in order to fix build issues related Xcode 12.5 and Apple Silicon (M1) machines.
https://github.com/facebook/react-native/blob/7dc0b5153e4eb91c333238a58fe8c75a47cb5f81/scripts/react_native_pods.rb#L644

I always try to keep in sync with the official `react-native` configuration, following https://react-native-community.github.io/upgrade-helper/?from=0.64.2&to=0.67.4.

Do you see any possible problem?

Reply:
Xcode 12.5 is not supported anymore by Apple (you cannot submit builds to the Store with Xcode <13). Also what it seems to do is upgrading the minimum deploy target to iOS 11 (that is something we have already made with #3928). If we don't have any specific problem related to this we could avoid it.

Reply:
Thanks for the insight, I agree with you, removed in https://github.com/pagopa/io-app/pull/3915/commits/ba4179aae2304e81543284d84849323979bbdc84